pokemongo_bot/__init__.py

```diff
@@ -295,7 +295,10 @@ def print_player_data(self, player):
         stardust = player['currencies'][1].get('amount', '0')
 
         logger.log('[#]')
-        logger.log('[#] Username: {username}'.format(**player))
+        if self.config.ci is False:
```

Comment:
Eh, fair enough, though there's a lot of other information that can be used to identify the test bot anyway.